### PR TITLE
heroku: FIX for node not found  https://github.com/NixOS/nixpkgs/issu…

### PR DESCRIPTION
**Pull request by kamilchm**
###### Things done:
- [x] Tested using sandboxing (`nix-build --option build-use-chroot true` or [nix.useChroot](http://nixos.org/nixos/manual/options.html#opt-nix.useChroot) on NixOS)
- [x] Built on platform(s): NixOS
- [ ] Tested compilation of all pkgs that depend on this change using `nix-shell -p nox --run "nox-review wip"`
- [x] Tested execution of all binary files (usually in `./result/bin/`)
- [x] Fits [CONTRIBUTING.md](https://github.com/NixOS/nixpkgs/blob/master/.github/CONTRIBUTING.md).
###### More

Fixes issue https://github.com/NixOS/nixpkgs/issues/10298

cc @aflatter @mirdhyn

### CODE REVIEW

File: pkgs/development/tools/heroku/default.nix [hunk 1]
```diff
@@ -17,10 +17,13 @@ stdenv.mkDerivation rec {
     sha256 = "1d472vm37lx5nyyaymjglavisb1mkyzbjglzjp5im7wjfifvsd29";
   };
 
+  phases = [ "unpackPhase" "installPhase" "fixupPhase" ];
```
Comment (joachifm) [inline pkgs/development/tools/heroku/default.nix]: Out of curiosity, which phase is causing a problem here?

Comment (aflatter) [inline pkgs/development/tools/heroku/default.nix]: Yes, same question here :-)

Comment (globin) [inline pkgs/development/tools/heroku/default.nix]: It will probably be `buildPhase` failing, because of nothing to run.

Comment (joachifm) [inline pkgs/development/tools/heroku/default.nix]: If I'm reading the docs correctly, `buildPhase` should do nothing unless a makefile exists. You may set `dontBuild` or `buildPhase = ":"` to specifically disable it. A little cleaner, IMO, but no big deal.

Comment (kamilchm) [inline pkgs/development/tools/heroku/default.nix]: I wanted to clean build logs:
```
configuring
grep: : No such file or directory
grep: : No such file or directory
no configure script, doing nothing
```
I can remove this line.

Comment (joachifm) [inline pkgs/development/tools/heroku/default.nix]: Please do

**Comment by joachifm** [comment]
I restarted the travis build. This looks sensible to me, feel free to ping me when you're happy to merge it.

PS: when you update the PR, subscribers are not notified.

**Comment by aflatter** [comment]
:tada:

**Pull request merged**